prog/target.go

```diff
@@ -31,6 +31,9 @@ type Target struct {
 	// SanitizeCall neutralizes harmful calls.
 	SanitizeCall func(c *Call)
 
+	// AnnotateCall annotates a syscall invocation in C reproducers.
```

Comment:
// AnnotateCall supplies a syscall invocation for C reproducers.
// Empty string means no annotation. The result is included into a comment.

Just to reduce duplication and give some hints to the implementors who could otherwise be confused about whether they are responsible for turning the result into a comment.